src/amber/router/params.cr

```diff
@@ -65,11 +65,12 @@ module Amber::Router
     end
 
     def to_h
-      query.to_h
-           .merge(form.to_h)
-           .merge(route)
-           .merge(json)
-           .merge(multipart)
+      params_hash = Hash(String, String | Array(String) | Nil).new
```

Comment:
Thought of memoizing this but it felt too risky, since the params can change withing the request

Reply:
The order here is important since we give form a higher priority it will override query and route params